homework/index.js

```diff
@@ -30,18 +30,70 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
+  function createLayout() {
+    const root = document.getElementById('root');
+    createAndAppend('select', root, { id: 'repo-select' });
+    createAndAppend('div', root, { id: 'body-container' });
+    createAndAppend('div', document.getElementById('body-container'), { id: 'repo-details' });
+    createAndAppend('div', document.getElementById('body-container'), { id: 'contributors' });
+
+  }
+
+  function getRepoDataFromOrgAndAddToDOM() {
+    const REPOS_URL = 'https://api.github.com/orgs/foocoding/repos?per_page=100';
+    fetchJSON(REPOS_URL, (err, arrayOfRepoData) => { 
       if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
+        alert(err.message);
       } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
+        arrayOfRepoData.sort((a, b) => (a.name.toLowerCase() > b.name.toLowerCase()) ? 1 : -1);
```

Comment:
Another way to do this is by using [`localeCompare`](https://kapeli.com/dash_share?docset_file=JavaScript&docset_name=JavaScript&path=developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare.html&platform=javascript&repo=Main&source=developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare), it will work better if you're using for example å in the text you're comparing:

```javascript
arrayOfRepoData.sort((a, b) => a.name.localeCompare(b.name);
```